app/tests/v1/test_user.py

```diff
@@ -33,4 +34,5 @@ def test_login_wrong_details(self):
 			content_type='application/json')
 		self.assertEqual(response.status_code, 404)
 		result = json.loads(response.data.decode())
-		self.assertEqual(result['message'], "User not registered")
\ No newline at end of file
+		self.assertEqual(result['message'], "User not registered")
+
```

Comment:
Great job testing a response to a user who is not registered.

Reply:
Thank You